OnServiceChange/handler.ts

```diff
@@ -178,16 +179,42 @@ export const log = (d: unknown): void => {
   throw new Error(`To be implement ${d}`);
 };
 
-export const onInvalidDocument = (d: unknown): T.Task<void> => {
+export const onInvalidDocument = (
```

Comment:
There is a 
```typescript
throw new Error(`To be implement ${JSON.stringify(d, null, 2)}`);
``` 
left at the end of this function.

Reply:
Yes, it's time to remove it. Fix with be4fe71